code/modules/mob/living/simple_animal/friendly/cat.dm

```diff
@@ -73,6 +73,18 @@
 	maxbodytemp = T0C + 40
 	held_state = "spacecat"
 
+/mob/living/simple_animal/pet/cat/breadcat
+	name = "Jason"
+	desc = "It's a cat... with a bread!"
+	gender = MALE
+	icon_state = "breadcat"
+	icon_living = "breadcat"
+	icon_dead = "breadcat_dead"
+	collar_type = null
+	unique_pet = TRUE
+	held_state = "breadcat"
+	butcher_results = list(/obj/item/reagent_containers/food/snacks/meat/slab = 2, /obj/item/organ/ears/cat = 1, /obj/item/organ/tail/cat = 1, /obj/item/food/breadslice/plain = 1)
```

Comment:
looks like you can make a bread cat, kill it, and then butcher it to convert kitty ears back into their cat ears+cat tail components (which might not have even existed in the first place if you just bought those kitty ears from an autodrobe or the like).

Reply:
Yes, I am aware of this.